src/Features/CSharp/Portable/ExtractLocalMethod/CSharpExtractLocalMethodCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CodeRefactorings.ExtractMethod;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
+using Microsoft.CodeAnalysis.ExtractMethod;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.ExtractLocalMethod
```

Comment:
as painful as it will be, these should all be updated to be 'ExtractLocalFunction'.  'local function' is what the langauge (and our docs) call these uys.